src/init/init.ts

```diff
@@ -156,12 +156,11 @@ function getDisplayName(generatorName: string) {
   // -------------------------------------------------------------------
   // (generator-)?           | Grp 1; Match "generator-"; Optional
   // (polymer-init)?         | Grp 2; Match "polymer-init-"; Optional
-  // (polymer-starter-kit-)? | Grp 3; Match "polymer-starter-kit-"; Optional
```

Comment:
I would but the commits were squashed so they don't exist in the branch or anywhere I can find them in our repo. I've mimic'd reverting the commit though, so this should be fine.